app/assets/v2/css/sidebar_search.css

```diff
@@ -106,6 +107,7 @@
     margin-left: 10px;
     margin-right: 10px;
     padding: 20px 20px 10px;
+    width: 100%;
```

Comment:
this file is shared on tribes and other part of the site I think should be better instead just use w-100 class on the html to be safe is not going to break nothing on the other pages

Reply:
Yeah this is probably a good idea @chibie - we often find when we change css thats used on multiple pages some visual errors someplace else weeks later... 😆

Reply:
oh right! i'll fix this in a bit

Reply:
@octavioamu done!